service/pom.xml

```diff
@@ -33,7 +33,7 @@
 
   <properties>
     <!-- the following property is used in the bash script as well, don't remove it! -->
-    <jetty.runner.version>9.4.14.v20181114</jetty.runner.version>
+    <jetty.runner.version>10.0.0-alpha0</jetty.runner.version>
```

Comment:
The latest stable version is `9.4.20.v20190813`, can we use that instead of the pre-release version?